### PR TITLE
Remove unnecessary async

### PR DESCRIPTION
**Pull request by halter73**

### CODE REVIEW

File: uController.SourceGenerator/CodeGenerator.cs [hunk 1]
```diff
@@ -89,11 +88,11 @@ public string Generate()
             var ctors = _model.HandlerType.GetConstructors();
             if (ctors.Length > 1 || ctors[0].GetParameters().Length > 0)
             {
-                WriteLine($"private readonly {S(typeof(ObjectFactory))} _factory = {S(typeof(ActivatorUtilities))}.CreateFactory(typeof({S(_model.HandlerType)}), {S(typeof(Type))}.EmptyTypes);");
+                WriteLine($"private readonly {typeof(ObjectFactory)} _factory = {typeof(ActivatorUtilities)}.CreateFactory(typeof({S(_model.HandlerType)}), {typeof(Type)}.EmptyTypes);");
```
Comment (davidfowl) [inline uController.SourceGenerator/CodeGenerator.cs:91]: Why this change?

File: Samples/Samples.csproj [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```
Comment (davidfowl) [inline Samples/Samples.csproj:4]: Is this required?

Comment (halter73) [reply Samples/Samples.csproj:4]: I did it for easier benchmarking.

File: uController.SourceGenerator/CodeGenerator.cs (hunk 1 above)
Comment (halter73) [reply uController.SourceGenerator/CodeGenerator.cs:91]: I just noticed the `S()` was unnecessary for the types I removed it from. I left `S()` for user-provided types because I'm guessing it's still necessary for that (but I'm not sure).

Comment (davidfowl) [reply uController.SourceGenerator/CodeGenerator.cs:91]: Add them all back 😄
